app/main.py

```diff
@@ -2,24 +2,73 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.protected_name = "_" + name
 
-class Visitor:
-   pass
+    def __get__(self, instance: str, owner: str) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: str, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError
+        setattr(instance, self.protected_name, value)
 
 
 class SlideLimitationValidator(ABC):
-    pass
+
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    weight = IntegerRange(20, 50)
+    height = IntegerRange(80, 120)
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(14, 60)
+    weight = IntegerRange(50, 120)
+    height = IntegerRange(120, 220)
+
+
+class Visitor:
+
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class Slide:
-    pass
+    def __init__(self, name: str,
+                 limitation_class: SlideLimitationValidator
+                 ) -> None:
```

Comment:
Thanks, because before I couldn't understand why Pycharm gave me an error here: "elf.limitation_class(visitor.age, visitor.weight, visitor.height) - not callable"